qemu/tests/cfg/viorng_in_use.cfg

```diff
@@ -26,12 +26,13 @@
     Linux:
         read_rng_cmd  = "dd if=/dev/random bs=10 count=10 2>/dev/null|hexdump"
         driver_verifier_cmd = "cat /sys/devices/virtual/misc/hw_random/rng_current"
-        target_process = rngd
+        target_process = dd
```

Comment:
dd might not be the proper target_process, for dd quit very soon, then the check_bg_running always returns false, and test (driver_in_use) failed. 
Also, please do a regression run for driver_in_use test.

Reply:
We use target_process to check the background stress thread status, for Linux, it's dd. So we should check dd rather than rngd.
And the dd command running in background is Line 50, "dd if=/dev/random bs=10 count=1000000 2>/dev/null|hexdump". It lasts enough long time for the foreground tests, like stop_continue/reboot, etc.
Thanks!